services/src/kilda-core/kilda-model/src/main/java/org/openkilda/utils/ICookieEnumField.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.utils;
+
+public interface ICookieEnumField {
```

Comment:
I see 2 issues here:
- Not obvious the purpose of the interface along with lack of javadoc.
- "I" prefix is against the code convention.